src/common.jl

```diff
@@ -41,6 +41,8 @@ const CONSTANT = Float64
 const LIM_TOL = 1e-6
 const XY_COORDS = @NamedTuple{x::Float64, y::Float64}
 
+const RNG_SEED = get(ENV, "SIENNA_RNG_SEED", 2017)
```

Comment:
Can you give an example of how setting the environment variable will be used? The value will be set the first time the code is compiled.